nvfuser/nvfuser_version.py

```diff
@@ -24,7 +24,7 @@ def v():
     def __init__(self, cls_name: str) -> None:
         self._cls_name = cls_name
 
-    def get_cls(self):
+    def get_cls(self) -> Any:
```

Comment:
I fixed this one missing return type annotation to try and trigger a mypy run over this file. Locally when I run `lintrunner nvfuser/nvfuser_version.py` I get a number of other errors.